src/main/kotlin/Bootstrap.kt

```diff
@@ -0,0 +1,71 @@
+package leia
+
+import leia.http.KtorServer
+import leia.http.ServerFactory
+import leia.logic.Resolver
+import leia.logic.ResolverAtom
+import leia.logic.SourceSpecsResolver
+import leia.registry.Registry
+import leia.registry.TomlRegistry
+import leia.sink.CachedSinkProviderFactory
+import leia.sink.DefaultSinkProviderFactory
+import leia.sink.Sink
+import leia.sink.SinkProvider
+import leia.sink.SinkProviderAtom
+import leia.sink.SpecSinkProvider
+import se.zensum.leia.config.SinkProviderSpec
+import se.zensum.leia.config.SourceSpec
+import se.zensum.leia.config.TomlConfigProvider
+import se.zensum.leia.getEnv
+
+fun run(sf: ServerFactory, resolver: Resolver, sinkProvider: SinkProvider) {
+    val res = sf.create(resolver, sinkProvider)
+}
```

Comment:
This seems a bit odd. Is this really what is intended? Why not just run `sf.create(resolver, sinkProvider)` instead of calling this function? `res` is assigned to the result of `sf.create` but not returned, is this intentional? Or maybe this function is not completed yet?